config/environments/development.rb

```diff
@@ -27,7 +29,21 @@
   end
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+  # actually send emails
+  config.action_mailer.perform_deliveries = true
+
+  config.action_mailer.delivery_method = :smtp
```

Comment:
For development purposes, could we use `:file` instead, or is that to complicated to test locally? http://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration

Reply:
I'm against that. I was pretty happy to see real emails in my personal mail program even in development mode. With this setting, one can also locally verify that the smtp connection is set up correctly.